autopkg/autopkg_tools.py

```diff
@@ -12,8 +12,8 @@
 
 
 DEBUG = False
-SLACK_WEBHOOK = os.environ.get("SLACK_WEBHOOK_TOKEN", None)
-MUNKI_REPO = os.path.join(os.environ["GITHUB_WORKSPACE"], "munki_repo")
+SLACK_WEBHOOK = os.environ.get("SLACK_WEBHOOK_TOKEN", "https://hooks.slack.com/services/CHANGE_ME")
```

Comment:
This should be none. The slack notification function gracefully handles none, letting the operator know why no slack message was sent. This will lead to an actual error.

Reply:
None leads to an exception on line 228 because you can't make a POST to None.

Reply:
You're right - there should be a return on line 200. I'll commit that to this branch.